2021-11-26 02:00:44,453 INFO    MainThread:47969 [wandb_setup.py:_flush():71] setting env: {}
2021-11-26 02:00:44,453 INFO    MainThread:47969 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-26 02:00:44,453 INFO    MainThread:47969 [wandb_init.py:_log_setup():371] Logging user logs to /opt/ml/code/wandb/run-20211126_020044-9mxcrzv4/logs/debug.log
2021-11-26 02:00:44,453 INFO    MainThread:47969 [wandb_init.py:_log_setup():372] Logging internal logs to /opt/ml/code/wandb/run-20211126_020044-9mxcrzv4/logs/debug-internal.log
2021-11-26 02:00:44,453 INFO    MainThread:47969 [wandb_init.py:init():404] calling init triggers
2021-11-26 02:00:44,453 INFO    MainThread:47969 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2021-11-26 02:00:44,453 INFO    MainThread:47969 [wandb_init.py:init():449] starting backend
2021-11-26 02:00:44,454 INFO    MainThread:47969 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-26 02:00:44,464 INFO    MainThread:47969 [backend.py:ensure_launched():199] starting backend process...
2021-11-26 02:00:44,472 INFO    MainThread:47969 [backend.py:ensure_launched():205] started backend process with pid: 48005
2021-11-26 02:00:44,473 INFO    MainThread:47969 [wandb_init.py:init():458] backend started and connected
2021-11-26 02:00:44,475 INFO    MainThread:47969 [wandb_init.py:init():519] updated telemetry
2021-11-26 02:00:44,476 INFO    MainThread:47969 [wandb_init.py:init():549] communicating current version
2021-11-26 02:00:45,308 INFO    MainThread:47969 [wandb_init.py:init():554] got version response 
2021-11-26 02:00:45,308 INFO    MainThread:47969 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-26 02:00:45,524 INFO    MainThread:47969 [wandb_init.py:init():592] starting run threads in backend
2021-11-26 02:00:50,461 INFO    MainThread:47969 [wandb_run.py:_console_start():1816] atexit reg
2021-11-26 02:00:50,461 INFO    MainThread:47969 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-26 02:00:50,461 INFO    MainThread:47969 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-26 02:00:50,464 INFO    MainThread:47969 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-26 02:00:50,464 INFO    MainThread:47969 [wandb_init.py:init():619] run started, returning control to user process
2021-11-26 02:10:56,703 INFO    MainThread:47969 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-26 02:10:56,703 INFO    MainThread:47969 [wandb_run.py:_restore():1758] restore
2021-11-26 02:10:59,362 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 507
}

2021-11-26 02:10:59,573 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 507
}

2021-11-26 02:10:59,828 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 43963
}

2021-11-26 02:10:59,930 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 47872
}

2021-11-26 02:11:00,038 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 507
  total_bytes: 47872
}

2021-11-26 02:11:00,140 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 47872
  total_bytes: 47872
}

2021-11-26 02:11:00,243 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 47872
  total_bytes: 47872
}

2021-11-26 02:11:00,345 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 47872
  total_bytes: 47872
}

2021-11-26 02:11:00,447 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 47872
  total_bytes: 47872
}

2021-11-26 02:11:00,549 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 47872
  total_bytes: 47872
}

2021-11-26 02:11:00,652 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 47872
  total_bytes: 47872
}

2021-11-26 02:11:01,113 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 47872
  total_bytes: 47872
}

2021-11-26 02:11:01,594 INFO    MainThread:47969 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 47872
  total_bytes: 47872
}
local_info {
}

2021-11-26 02:11:02,783 INFO    MainThread:47969 [wandb_run.py:_append_files():2184] logging synced files
